README.md

```diff
@@ -338,27 +338,35 @@ Once complete your base AWS resources should be in place
 
 * [kubectl](https://kubernetes.io/docs/user-guide/prereqs/)
 * [Kops](https://github.com/kubernetes/kops)
-* jq
-* yq
+* Python 3.6+
 
-(On macOS you can: `brew install kubectl kops jq yq`)
+(On macOS you can: `brew install kubectl kops python3`)
 
-2. Copy an existing cluster config:
-```
-cp -R infra/kops/clusters/alpha infra/kops/clusters/$ENVNAME
-```
+2. Create a values YAML file for Kops:
+  Most values required by Kops can be obtained from Terraform, but there is currently
+  one exception - the OIDC client ID used for cluster authentication. For this, create
```

Comment:
There are several client IDs kicking around - to be ultra clear, can it be described as the "Auth0 'kubectl-oidc' application's client ID"